tests/integration.rs

```diff
@@ -92,6 +93,53 @@ async fn test_read_stream_events(client: &Client) -> Result<(), Box<dyn Error>>
     Ok(())
 }
 
+async fn test_metadata(client: &Client) -> Result<(), Box<dyn Error>> {
+    let stream_id = fresh_stream_id("metadata");
+    let events = generate_events("metadata-test".to_string(), 5);
+
+    let _ = client
+        .append_to_stream(stream_id.as_str(), &Default::default(), events)
+        .await?;
+
+    let expected = StreamMetadataBuilder::new()
+        .max_age(2)
```

Comment:
[Failing CI test](https://github.com/EventStore/EventStoreDB-Client-Rust/pull/62/checks?check_run_id=2628545113#step:3:443) : `max_age()` takes Duration

```suggestion
        .max_age(Duration::from_secs(2))
```

Reply:
Done.